scripts/build/parse_syscalls.py

```diff
@@ -101,6 +102,11 @@ def analyze_headers(include_dir, scan_dir, file_list):
                                                    'common.h'))):
                     continue
 
+                # On windows, change to posix style paths to avoid duplicate
+                # entries with differing folder separators.
+                if os.name == 'nt':
+                    path = PureWindowsPath(path).as_posix()
```

Comment:
any reason this needs to be tied to the os.

Why not just use PurePath and have identical code across all platforms ?
```suggestion
                PurePath(os.path.normpath(path)).as_posix())
```

Reply:
Fixed